app/src/main/java/org/apache/fineract/ui/adapters/CustomerAdapter.java

```diff
@@ -67,7 +67,12 @@ public void onBindViewHolder(ViewHolder holder, int position) {
                 customer.getIdentifier()), holder.ivCustomerImage,
                 R.drawable.ic_account_circle_black_24dp);
 
-        StatusUtils.setCustomerStatus(customer.getCurrentState(), holder.ivStausIndicator, context);
+        if (customer.getCurrentState() != null) {
+            StatusUtils.setCustomerStatus(customer.getCurrentState(), holder.ivStausIndicator,
```

Comment:
wrong, It can't be null in any case

Reply:
It can be now, this same adapter is being used in the draft customers section where the draft customer does not have a state.